cluster.go

```diff
@@ -11,6 +11,8 @@ import (
 	"time"
 )
 
+const defaultDriverName = "ScyllaDB GoLang Driver"
```

Comment:
Isn't "Go" official language name?

Reply:
Yes, but it's similarity to the verb makes everybody use `golang` to avoid confusion, even oficial repo is named [golang/go](https://github.com/golang/go)

Reply:
Maybe we should use "gocql" to identify it's coming from this driver.
There is another golang driver and one could use that, so maybe we should differentiate it by name.

Reply:
How do we name `scylla-go-driver` then ?

Reply:
I would call it "ScyllaDB gocql Driver".
BTW, how does it appear now in system.clients?

Reply:
My proposal is this:
`scylladb/gocql`: `ScyllaDB GoLang Driver`
`scylladb/scylla-go-driver`: `ScyllaDB NextGen GoLang Driver`

Reply:
> I would call it "ScyllaDB gocql Driver". BTW, how does it appear now in system.clients?

It does not provide it's name

Reply:
How about `ScyllaDB gocql Driver` for gocql and `ScyllaDB NextGen Go Driver`? Imo gocql is an established and we should use it here

Reply:
Let's vote:
1. :rocket: `ScyllaDB gocql Driver` / `ScyllaDB NextGen Go Driver`
2. :heart: `ScyllaDB GoCQL Driver` / `ScyllaDB NextGen Go Driver`
3. :eyes:  `ScyllaDB GoLang Driver` / `ScyllaDB NextGen GoLang Driver`

Feel free to add your variant

Reply:
It's kinda weird start with uppercase and suddently drop to lowercase...

Reply:
@roydahan , @sylwiaszunejko join in to vote at https://github.com/scylladb/gocql/pull/228#discussion_r1704493683

Reply:
@dkropachev lets put aside Scylla-go-driver, I don't know if it's "NextGen" and AFAIC it's not our recommended driver or supported one.

Reply:
changed it to `ScyllaDB GoCQL Driver`